### ISSUE TITLE
Hyperparms for pt3

### ISSUE THREAD

**Issue by AlexKer** [mikeizbicki/cmc-csci181#13]
It'll be great if you could provide the full list of hyper parameters that were used for the part 3 video tutorial for the GRU. I'm currently trying to replicate the results in tensorboard such that I can generate comparable strings.

**Comment by mikeizbicki** [comment]
My initial training is done with
```
$ python3 names.py --train --model=gru --learning_rate=1e-1 --batch_size=10 --gradient_clipping --hidden_layer_size=128 --num_layers=1 --num_samples=100000
```
and then I decay the learning rate by 10 twice.
